src/Template/Pages/Dashboard/index.twig

```diff
@@ -5,57 +5,51 @@
     <div class="dashboard">
         <section class="dashboard-section">
             <div class="dashboard-items">
-            {% for name, module in modules if not in_array(name, ['trash', 'objects', 'users', 'folders', 'publications']) %}
-                {% set title = Layout.typeLabel(module.label|default(name)|humanize) %}
-                {% set class = 'dashboard-item has-background-module-%s'|format(name) %}
-                {% if module.hints.multiple_types %}
-                    {% set class = class ~ ' icon-th-large-1' %}
-                {% endif %}
+            {% for name, module in modules if not in_array(name, ['trash', 'users']) %}
 
-                {% if module.route %}
-                    {% set url = Url.build(module.route) %}
-                {% else %}
-                    {% set url = Url.build({ '_name': 'modules:list', 'object_type': name, 'plugin': null }) %}
-                {% endif %}
-                <a href="{{ url }}" class="{{ class }}">
-                    <span>{{ title }}</span>
-                </a>
-            {% endfor %}
+                {% if name == 'objects' %}
```

Comment:
I suggest this change, since in every module `label` and `class` can be defined in configuration, we are allowed to override them in every module.

```twig 
                {# Special cases - set defaults on objects, publications, folders #}

                {% if name == 'objects' %}
                    {% set module = {'label': 'All objects'}|merge(module) %}
                {% elseif name == 'publications' %}
                    {% set module = {'class': 'icon-globe-1'}|merge(module) %}
                {% elseif name == 'folders' %}
                    {% set module = {'class': 'icon-folder'}|merge(module) %}
                {% endif %}

                {% set title = Layout.typeLabel(module.label|default(name)|humanize) %}

                {% set class = 'dashboard-item has-background-module-%s %s'|format(name, module.class|default('')) %}
                {% if module.hints.multiple_types and not module.class %}
                    {% set class = class ~ ' icon-th-large-1' %}
                {% endif %}

```
then unchanged from line 
same logic could be applied in modules below, we could also use an element for this part